.github/workflows/php-static-analysis.yml

```diff
@@ -31,12 +31,37 @@ jobs:
         uses: inpsyde/reusable-workflows/.github/workflows/lint-php.yml@main
         if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run lint only')) }}
         with:
-            PHP_MATRIX: '["7.2", "7.3", "7.4", "8.0", "8.1", "8.2"]'
+            PHP_MATRIX: '["8.0", "8.1", "8.2"]'
 
-    coding-standards-analysis-php:
+    coding-standards-analysis-php-80:
         if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run PHPCS only')) }}
         uses: inpsyde/reusable-workflows/.github/workflows/coding-standards-php.yml@main
+        with:
+            PHP_VERSION: '8.0'
+    coding-standards-analysis-php-81:
+        if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run PHPCS only')) }}
+        uses: inpsyde/reusable-workflows/.github/workflows/coding-standards-php.yml@main
+        with:
+            PHP_VERSION: '8.1'
+    coding-standards-analysis-php-82:
+        if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run PHPCS only')) }}
+        uses: inpsyde/reusable-workflows/.github/workflows/coding-standards-php.yml@main
+        with:
+            PHP_VERSION: '8.2'
 
-    static-code-analysis-php:
+    static-code-analysis-php-80:
         if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run Psalm only')) }}
         uses: inpsyde/reusable-workflows/.github/workflows/static-analysis-php.yml@main
+        with:
+            PHP_VERSION: '8.0'
+    static-code-analysis-php-81:
+        if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run Psalm only')) }}
+        uses: inpsyde/reusable-workflows/.github/workflows/static-analysis-php.yml@main
+        with:
+            PHP_VERSION: '8.1'
+    static-code-analysis-php-82:
+        if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run Psalm only')) }}
+        uses: inpsyde/reusable-workflows/.github/workflows/static-analysis-php.yml@main
+        with:
+            PHP_VERSION: '8.2'
+
```

Comment:
Why not using a single with a matrix instead of multiple jobs?

Reply:
done